PWGHF/Tasks/taskX.cxx

```diff
@@ -49,11 +49,16 @@ struct TaskX {
 
   Configurable<int> d_selectionFlagX{"d_selectionFlagX", 1, "Selection Flag for X"};
   Configurable<double> cutYCandMax{"cutYCandMax", -1., "max. cand. rapidity"};
+  Configurable<bool> d_modeXToJpsiToMuMuPiPi{"d_modeXToJpsiToMuMuPiPi", false, "Perform Jpsi to mu+mu- analysis"};
   Configurable<std::vector<double>> bins{"pTBins", std::vector<double>{hf_cuts_x_tojpsipipi::pTBins_v}, "pT bin limits"};
 
   void init(o2::framework::InitContext&)
   {
-    registry.add("hMass", "3-prong candidates;inv. mass (J/#psi #pi+ #pi-) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{200, 3., 4.}, {(std::vector<double>)bins, "#it{p}_{T} (GeV/#it{c})"}}});
+    if (d_modeXToJpsiToMuMuPiPi) {
+      registry.add("hMass", "3-prong candidates;inv. mass (J/#psi(#mu+#mu-) #pi+ #pi-) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{200, 3., 4.}, {(std::vector<double>)bins, "#it{p}_{T} (GeV/#it{c})"}}});
+    } else {
+      registry.add("hMass", "3-prong candidates;inv. mass (J/#psi(e+e-) #pi+ #pi-) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{200, 3., 4.}, {(std::vector<double>)bins, "#it{p}_{T} (GeV/#it{c})"}}});
+    }
```

Comment:
The only difference seems to be the axis label, which can be defined on one line and used directly in the histogram line.

Reply:
It was a leftover of some past checks, I have removed that